src/index.js

```diff
@@ -5,6 +5,8 @@ import Routes from './routes/index';
 import store from './Store';
 import './assets/style.scss';
 import './assets/css/style.css';
+import '../node_modules/argon-design-system-free/assets/css/argon.css';
+import '../node_modules/argon-design-system-free/assets/css/argon.min.css';
```

Comment:
Why are you importing both the minified version and the one that's not minified? Isn't this repetition?

Reply:
The minified version has some extra css argon features @RrNn .

Reply:
Really? This sounds weird because I assume they minified what you're importing on line **8** to get what you're importing on line **9**. @kdaglas do you confirm this?

Reply:
I have gone through the minified CSS, and the lines of the css I needed are also in the main css though there some additional which I can just add to my custom CSS, for some reason I thought that I need them but I have just seen that I can just replicate them in my custom CSS. for that reason, i think she can remove that import because at this point it is becoming a repetition, sorry @d-rita for the inconvenience

Reply:
@d-rita, you can now happily remove one of the imports, preferably the un-minified one.

Reply:
I agree

Reply:
she should remove the minified one, not the un-minified one